# Solaris-specific system paths.
# Our bash configuration lives in /etc/bash/inputrc.
# Not for upstream.

--- lib/readline/rlconf.h
+++ lib/readline/rlconf.h
@@ -43,7 +43,7 @@ /* The next-to-last-ditch effort file na
 #define DEFAULT_INPUTRC "~/.inputrc"
 
 /* The ultimate last-ditch filename for an init file -- system-wide. */
-#define SYS_INPUTRC "/etc/inputrc"
+#define SYS_INPUTRC "/etc/bash/inputrc"
 
 /* If defined, expand tabs to spaces. */
 #define DISPLAY_TABS
--- doc/bashref.info
+++ doc/bashref.info
@@ -7586,7 +7586,7 @@ putting commands in an "inputrc" file, c
 directory.  The name of this file is taken from the value of the shell
 variable 'INPUTRC'.  If that variable is unset, the default is
 '~/.inputrc'.  If that file does not exist or cannot be read, the
-ultimate default is '/etc/inputrc'.  The 'bind' builtin command can also
+ultimate default is '/etc/bash/inputrc'.  The 'bind' builtin command can also
 be used to set Readline keybindings and variables.  *Note Bash
 Builtins::.
 
@@ -8135,8 +8135,8 @@ test fails.
 '$include'
      This directive takes a single filename as an argument and reads
      commands and bindings from that file.  For example, the following
-     directive reads from '/etc/inputrc':
-          $include /etc/inputrc
+     directive reads from '/etc/bash/inputrc':
+          $include /etc/bash/inputrc
 
 
 File: bashref.info,  Node: Sample Init File,  Prev: Conditional Init Constructs,  Up: Readline Init File
@@ -8155,8 +8155,8 @@ # You can re-read the inputrc file with
      # Lines beginning with '#' are comments.
      #
      # First, include any system-wide bindings and variable
-     # assignments from /etc/Inputrc
-     $include /etc/Inputrc
+     # assignments from /etc/bash/inputrc
+     $include /etc/bash/inputrc
 
      #
      # Set various bindings for emacs mode.
--- doc/bash.1
+++ doc/bash.1
@@ -6114,11 +6114,11 @@ the test fails.
 .IP \fB$include\fP
 This directive takes a single filename as an argument and reads commands
 and bindings from that file.  For example, the following directive
-would read \fI/etc/inputrc\fP:
+would read \fI/etc/bash/inputrc\fP:
 .sp 1
 .RS
 .nf
-\fB$include\fP \^ \fI/etc/inputrc\fP
+\fB$include\fP \^ \fI/etc/bash/inputrc\fP
 .fi
 .RE
 .SS Searching
--- doc/bash.info
+++ doc/bash.info
@@ -7586,7 +7586,7 @@ putting commands in an "inputrc" file, c
 directory.  The name of this file is taken from the value of the shell
 variable 'INPUTRC'.  If that variable is unset, the default is
 '~/.inputrc'.  If that file does not exist or cannot be read, the
-ultimate default is '/etc/inputrc'.  The 'bind' builtin command can also
+ultimate default is '/etc/bash/inputrc'.  The 'bind' builtin command can also
 be used to set Readline keybindings and variables.  *Note Bash
 Builtins::.
 
@@ -8135,8 +8135,8 @@ test fails.
 '$include'
      This directive takes a single filename as an argument and reads
      commands and bindings from that file.  For example, the following
-     directive reads from '/etc/inputrc':
-          $include /etc/inputrc
+     directive reads from '/etc/bash/inputrc':
+          $include /etc/bash/inputrc
 
 
 File: bash.info,  Node: Sample Init File,  Prev: Conditional Init Constructs,  Up: Readline Init File
@@ -8155,8 +8155,8 @@ # You can re-read the inputrc file with
      # Lines beginning with '#' are comments.
      #
      # First, include any system-wide bindings and variable
-     # assignments from /etc/Inputrc
-     $include /etc/Inputrc
+     # assignments from /etc/bash/inputrc
+     $include /etc/bash/inputrc
 
      #
      # Set various bindings for emacs mode.
